src/WebJobs.Script/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Azure.WebJobs.Script.Abstractions.Description;
+using Microsoft.Azure.WebJobs.Script.Description;
+
+namespace Microsoft.Azure.WebJobs.Script.Extensions
+{
+    public static class FunctionMetadataExtensions
```

Comment:
When we do the pass over the public surface area of the new Types, some of the helpers like FunctionMetadata.IsHttpInAndOutFunction can move here as well I think